pkg/cmd/kafka/create/create.go

```diff
@@ -282,6 +264,11 @@ func runCreate(opts *options) error {
 
 	}
 
+	if userInstanceType.BillingModel == "standard" || userInstanceType.BillingModel == "marketplace" {
```

Comment:
There are no other billing models and if they would be in future we should still pass them. I think this is redundant.